lib/views/past_student.dart

```diff
@@ -48,8 +49,8 @@ class PastStudentState extends State<PastStudent> {
         title: const Text('Past Students'),
         actions: [
           IconButton(
-            icon: const Icon(Icons.delete),
-            onPressed: _deleteAllEntries,
+            icon: const Icon(Icons.cloud),
+            onPressed: _sentDataToCloud,
           ),
```

Comment:
nit : change icon from delete to cloud

cloud icon clearly shows what does the icon button do